packages/sysutils/busybox/scripts/init

```diff
@@ -1106,12 +1106,19 @@ for i in $STORAGE $FLASH ; do
     fi
   fi
 done
+
+# create /run filesystem with options used by systemd
+/usr/bin/busybox mount -t tmpfs -o mode=755,size=20%,nr_inodes=800k,nosuid,nodev,strictatime tmpfs /sysroot/run
```

Comment:
Please move to the top of the file where other filesystems are mounted and apply to `/run`. Then below use `mount --move` to move it to `/sysroot/run`.

Also, are all these options really needed? Will systemd apply those options by itself?

Reply:
I've moved /run creation to top as suggested - that might be helpful for future refactoring, we should probably move some of the marker files from /dev to /run at some point.

I can't tell if all of these options are really required (size limiting is probably more useful for machines with tons of RAM), but as systemd will just use the previously mounted /run fs as is (without adjusting any mount options) I didn't see a reason to deviate from systemd's defaults - this makes sure the options are identically to what we had be using before this PR.

see https://github.com/systemd/systemd/blob/v247/src/core/mount-setup.c#L87-L88
```
        { "tmpfs",       "/run",                      "tmpfs",      "mode=755" TMPFS_LIMITS_RUN,               MS_NOSUID|MS_NODEV|MS_STRICTATIME,
          NULL,          MNT_FATAL|MNT_IN_CONTAINER },
```
and https://github.com/systemd/systemd/blob/v247/src/shared/mount-util.h#L20-L22
```
/* On an extremely small device with only 256MB of RAM, 20% of RAM should be enough for the re-execution of
 * PID1 because 16MB of free space is required. */
#define TMPFS_LIMITS_RUN             ",size=20%,nr_inodes=800k"
```